### PR TITLE
Included google() repository in gradle build

### PR DESCRIPTION
**Pull request by sudipbhandari126**
Fixes #<156> (it's a good idea to open an issue first for context and/or discussion)

I think it's advisable to have this info in the readme itself since it's required only for version 1.104 and upwards.

### CODE REVIEW

**Comment by sudipbhandari126** [comment]
@googlebot I fixed it.

**Comment by sudipbhandari126** [comment]
@athakor @elharo

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ If you are using Maven without Bom, Add this to your dependencies.
 ```
 If you are using Gradle, add this to your dependencies
 ```Groovy
+repositories {
+        google() //this is required from version 1.104 onwards
+        //other repositories
```
Comment (elharo) [inline README.md]: Do you need to add `central()` explicitly too?

File: README.md [hunk 2]
```diff
@@ -42,6 +42,11 @@ If you are using Maven without Bom, Add this to your dependencies.
 ```
 If you are using Gradle, add this to your dependencies
 ```Groovy
+repositories {
+        google() //this is required from version 1.104 onwards
```
Comment (elharo) [inline README.md]: comment can be removed. This won't be necessary in the near future.

File: README.md (hunk 1 above)
Comment (sudipbhandari126) [reply README.md]: Yes. I added the additional part that would be required

File: README.md (hunk 2 above)
Comment (sudipbhandari126) [reply README.md]: Sure. (when I make this change.. is it okay to amend and force push or a separate commit)?

Comment (sudipbhandari126) [reply README.md]: @elharo done !!

Comment (elharo) [reply README.md]: Just make multiple commits and push again. I'll squash when it's merged.

Comment (sudipbhandari126) [reply README.md]: @elharo done

File: README.md (hunk 1 above)
Comment (elharo) [reply README.md]: you need to put central here too

Comment (sudipbhandari126) [reply README.md]: done

Comment (sudipbhandari126) [reply README.md]: @elharo should I make any more changes?

**Pull request merged**